src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -31,36 +30,19 @@ namespace winrt
     using IInspectable = Windows::Foundation::IInspectable;
 }
 
-static constexpr std::wstring_view StartupTaskName = L"StartTerminalOnLoginTask";
-// clang-format off
+////////////////////////////////////////////////////////////////////////////////
+// Error message handling. This is in this file rather than with the warnings in
+// TerminalWindow, because the error text might also just be a serialization

```

Comment:
nit: reading this I don't understand the connection between being "ust a serialization error" and "therefore being in this file"